internal/kibana/ingestmanager/client_agents.go

```diff
@@ -63,32 +64,51 @@ func (c *Client) AssignPolicyToAgent(a Agent, p Policy) error {
 	return nil
 }
 
+func (c *Client) getTotalAgentForPolicy(p Policy) (int, error) {
+	kuery := url.QueryEscape(fmt.Sprintf("fleet-agents.policy_id:\"%s\"", p.ID))
+	path := fmt.Sprintf("agents?kuery=%s", kuery)
+	statusCode, respBody, err := c.get(path)
+	if err != nil {
+		return -1, errors.Wrapf(err, "could not check agent status; API status code = %d; policy ID = %s; response body = %s", statusCode, p.ID, string(respBody))
```

Comment:
Should be okay to return `0` as the first return value here, yes?

Reply:
Generally speaking the Golang idiom is to return a data type's zero-value as the return value if there is an error also being returned. The expectation is that the caller will check for the error and ignore any other return values.